packages/compass-components/src/components/modals/modal.tsx

```diff
@@ -10,12 +10,38 @@ const contentStyles = css({
   padding: 0,
 });
 
+const modalFullScreenStyles = css({
+  '& > div': {
+    paddingTop: spacing[4],
+    paddingBottom: spacing[4],
+    paddingLeft: spacing[5],
+    paddingRight: spacing[5],
+    height: '100vh',
+    maxHeight: '100vh',
+  },
+});
+
+const contentFullScreenStyles = css({
+  width: '100%',
+  height: '100%',
+  maxHeight: '100%',
+  margin: 0,
+  alignSelf: 'stretch',
+  '& > div': {
+    height: '100%',
+    maxHeight: '100%',
+  },
+});
+
 function Modal({
   className,
   contentClassName,
   children,
+  fullScreen = false,
```

Comment:
This is a second full screen modal we have in the app now so felt appropriate to move this logic to the shared component